### PR TITLE
[WIP] Actuator board firmware

### PR DESCRIPTION
**Pull request by nuft**
I'm opening this PR to have a place for discussion.

### CODE REVIEW

File: uavcan_data_types/cvra/actuator/20150.Feedback.uavcan [hunk 1]
```diff
@@ -2,6 +2,6 @@
 # Feedback coming from actuator-firmware nodes
 #
 
-float16[2] pressure       # [Pascal]
+uint33[2]  pressure       # [Pascal]
```
Comment (nuft) [inline uavcan_data_types/cvra/actuator/20150.Feedback.uavcan:5]: A particular reason why uint33 or is this a typo?
In the code we're writing a uint32_t anyway.

Comment (antoinealb) [reply uavcan_data_types/cvra/actuator/20150.Feedback.uavcan:5]: Definitely a typo.

**Comment by antoinealb** [comment]
@nuft I would like to rebase and merge this, so that it does not diverge from the tree, and we can ensure it always build when changing libraries / build system / whatever. Ok for you ?

**Comment by nuft** [comment]
Agree, this will make things simpler to merge.

**Pull request merged**